Formula/hive.rb

```diff
@@ -1,16 +1,16 @@
 class Hive < Formula
   desc "Hadoop-based data summarization, query, and analysis"
   homepage "https://hive.apache.org"
-  url "https://www.apache.org/dyn/closer.cgi?path=hive/hive-1.2.1/apache-hive-1.2.1-bin.tar.gz"
-  sha256 "29d9780c4af887ef623bafe6a73ec6f1bea9759bbe31fb4aeeb5b0f68c4c9979"
+  url "https://www.apache.org/dyn/closer.cgi?path=hive/hive-2.0.0/apache-hive-2.0.0-bin.tar.gz"
+  sha256 "e854798954f48bdbc40235b1367287218464b62376d29dfcd2a328a0e7a184e6"
 
   bottle :unneeded
 
   depends_on "hadoop"
   depends_on :java
 
   def install
-    rm_f Dir["bin/ext/*.cmd", "bin/ext/util/*.cmd"]
+    rm_f Dir["bin/*.cmd, bin/ext/*.cmd", "bin/ext/util/*.cmd"]
```

Comment:
This looks wrong. The comma should be separating individual arguments to `Dir[]`, not be part of the first string argument. I suspect this should be instead:

```ruby
rm_f Dir["bin/*.cmd", "bin/ext/*.cmd", "bin/ext/util/*.cmd"]
```